lib/suspenders/app_builder.rb

```diff
@@ -400,6 +400,18 @@ def provide_deploy_script
       run "chmod a+x bin/deploy"
     end
 
+    def configure_automatic_deployment
+      deploy_command = <<-YML.strip_heredoc
+      deployment:
+        staging:
+          branch: master
+          commands:
```

Comment:
> I considered that but since the staging remote is set when the app is generated with the Heroku flag, I thought it was unnecessary.

That's locally though, not on the CI host. If the suspended app is generated with the Heroku flag, the deploy script would fail in Circle.